xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -4578,6 +4582,322 @@ void FixDimsForRaggedOffset(std::vector<int64_t>& dims, int max_reg_per_batch) {
   dims[0] *= max_reg_per_batch;
 }
 
+class ScoreModFunc {
```

Comment:
The following code looks complex enough to deserve some test coverage. Can you please add some?

Reply:
Added